obspy/io/nlloc/__init__.py

```diff
@@ -15,11 +15,47 @@
 
 Example
 -------
+If NonLinLoc location run was performed using one of the standard projections
+available in the NonLinLoc package, it is straightforward to read a NonLinLoc
+Hypocenter-Phase file into an ObsPy :class:`~obspy.core.event.Catalog` object:
 
-First, if NonLinLoc location run was performed in some custom coordinate system
-(as opposed to WGS84 with depth in meters down), we need to set up a coordinate
-conversion function to convert from the NonLinLoc location coordinates `x`,
-`y`, `z` to longitude, latitude and depth in kilometers.
+>>> from obspy import read_events
+>>> cat = read_events("/path/to/nlloc.hyp")  # doctest: +SKIP
+>>> print(cat)  # doctest: +SKIP
```

Comment:
Actually, you can run all those doctests, they should work just fine. I was just skipping the other doctests because they were relying on pyproj and that's not a dependency of ours (although we have it installed in Travis for testing now).